### DISCUSSION TITLE
this project is very cool

### DISCUSSION THREAD

**Discussion by exp1x1** [mat-sz/filedrop-web#24] [Q&A] [answered]
i know react and ts so how can i contribute ?

**Comment by mat-sz** [comment, marked as answer]
Hey, I'm currently in the middle of refactoring most of the codebase so can't really accept external help without making the entire thing too chaotic.

Thank you for the offer though, if anything comes to my mind I'll let you know.

**Reply by exp1x1** [reply]
ok, I am always happy to help open source tools that I use

**Reply by mat-sz** [reply]
@exp1x1 Would be great if you could test out the new version (just deployed) and see if there are any bugs (either visual or more serious).

**Reply by exp1x1** [reply]
sorry, i couldn't test because i was busy in job search, anything going on right now?
